.gitignore

```diff
@@ -1,4 +1,5 @@
 **/myvalues.yaml
+**/Chart.lock
```

Comment:
@CptSchnitz we should have `Chart.lock` in our charts, can we pass on this one? since pg is used for development purposes? we can't let Argo determine wether or not we use `Chart.lock`, @yossiz16 wdyt?

Reply:
`Chart.lock` isn't committed for now